deployment/docker-compose/template.core_backend.env

```diff
@@ -37,6 +37,11 @@ LITELLM_ENDPOINT="http://localhost:4000"
 #PGVECTOR_VECTOR_SIZE=1024
 
 #### Speech APIs ###############################################################
+# This variable controls whether the voice search endpoint is active (set to true) or inactive (set to false). Default is false.
+# If enabled, we default to using external services unless `CUSTOM_SPEECH_ENDPOINT` is set, in which case the custom hosted APIs will be used.
+# ENABLE_VOICE_SEARCH=True
+
+# if TOGGLE_VOICE is set to 'Custom' then make sure to also set the Environment variables mentioned below
```

Comment:
What is `TOGGLE_VOICE`, I don't see anywhere else. Probably that's what turned into `ENABLE_VOICE_SEARCH`

Reply:
You should probably change that in the docs. I saw it in `docs/components/voice-service/inded.md`